src/providers/http_provider.py

```diff
@@ -12,6 +13,11 @@
 logger = logging.getLogger(__name__)
 
 
+def _url_to_request_name_label(url: str) -> str:
+    """Remove all params from url and replace them with {param}"""
+    return '/'.join(['{param}' if ('0x' in part or part.isdigit()) else part for part in url.split('?')[0].split('/')])
```

Comment:
I'm not sure about it
URL path may have some credentials like api keys
I think we should add these metrics to the implementations, not a base class

Reply:
`url` arg already contains only API endpoints, without any credentials. Currently, we have only `KeysAPI` and `CL API `providers, which parent is `HttpProvider` and both of them have the same logic for working with `url`